flanker/addresslib/plugins/gmail.py

```diff
@@ -48,12 +55,12 @@ def validate(localpart):
     if not localpart:
         return False
 
-    slpart = localpart.replace('.', '')
-    lparts = slpart.split('+')
+    lparts = localpart.split('+')
     real_localpart = lparts[0]
+    stripped_localpart = real_localpart.replace('..', '')
```

Comment:
For the `stripped_localpart` length check, it's the length stripping all dot (.) from the local-part. You can fix that by changing this line to:

```
stripped_localpart = real_localpart.replace('.', '')
```